options/m_option.h

```diff
@@ -161,6 +161,8 @@ struct m_obj_list {
     void (*print_help_list)(struct mp_log *log);
     // Callback to print help for _unknown_ entries with "vf=entry=help"
     void (*print_unknown_entry_help)(struct mp_log *log, const char *name);
+    // Get lavfi filters for option-info/[av]f/choices.
+    const char **(*get_lavfi_filters)(void *talloc_ctx);
```

Comment:
Do you want to improve it or should I? But you clearly have an idea. It is internal API, so we can freely improve it anytime.

Reply:
Do you mean for `get_desc_iter` to return lavfi filters or all filters? Because if it only returns lavfi filters that's a confusing name, and if it returns all filters at that point we could just modify `get_desc` to also return lavfi filters and remove `print_help_list`, but I wasn't sure of the consequences of that since `init_obj_settings_list` assumes that `get_desc` only returns builtin filters.